sklearn/utils/_test_common/instance_generator.py

```diff
@@ -304,48 +316,93 @@ def _generate_pipeline():
         )
 
 
+INIT_PARAMS = {
```

Comment:
The name here is a bit confusing because we have now `TEST_PARAMS` and `INIT_PARAMS` and at a first glance this is not really easy to know why we have 2 dictionary.

Why the `INIT_PARAM` would not be enough to run all tests?

Reply:
I didn't want to change much behaviour with this PR. Merging the two would mean we'd be setting the parameters according to `TEST_PARAMS` in all tests, while we're not doing now. I'll see if anything fails if I merge them.